### DISCUSSION TITLE
What's the plan for React Server Components?

### DISCUSSION THREAD

**Discussion by borekb** [remix-run/remix#552] [Proposals]
Hi, congrats on the launch today!

Just curious, what's the plan (if any) for React Server Components? Remix seems to be tackling some of the same challenges as RSC so I wonder if it's more of an alternative to them or if, in the future, Remix will build on top of them?

Thanks and congrats again! 🎉

**Comment by sergiodxa** [comment]
I'm pretty sure Remix will use them internally in the future once they are stable. The great things is that us (the devs using Remix) will probably don't need need to think about them at all.
